charts/piraeus/templates/csi-controller-rbac.yml

```diff
@@ -93,7 +93,51 @@ roleRef:
   kind: ClusterRole
   name: external-provisioner-runner
   apiGroup: rbac.authorization.k8s.io
-
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: csi-provisioner-capacity-syncer
```

Comment:
Would `csi-capacity-syncer` do? The 2 `*er` words in the name make it a bit of a mouthful.